### ISSUE TITLE
[6.x] mmaps problem (no movement and wp bug)

### ISSUE THREAD

**Issue by et65** [TrinityCore/TrinityCore#14701]
Since ed75b0649add23e082976fa4e5d504bc0c312602 mmaps don't work and make all creatures movements bug : 
- creatures don't move to the attacker ;
- creatures don't make random movements ;
- when adding wp to a creature, they make random movement (like they try to follow the path, but they make very strange movements).

It happens on all maps (old maps < 1000 and new maps), it s due to mmaps because when I disable them, it works again.

Here the logs for one creature when it normally have to aggro me : 

```
++ PathGenerator::PathGenerator for GUID Full: 0x1C0004000003FAC000000000000004C9 Type: Creature Entry: 4075 Low: 1225
++ PathGenerator::CalculatePath() for GUID Full: 0x1C0004000003FAC000000000000004C9 Type: Creature Entry: 4075 Low: 1225
++ BuildPolyPath :: (startPoly == 0 || endPoly == 0)

++ BuildShortcut :: making shortcut
```

All creatures have this same output.

PS: Of course I've re-extracted maps, vmaps and mmaps since the commit.

**Comment by DDuarte** [comment]
Try #14702

**Comment by et65** [comment]
I'm re-extracting the mmaps. I think it will be good, I'll give you a confirmation when it will be finished.

**Comment by et65** [comment]
https://github.com/TrinityCore/TrinityCore/pull/14702 fix the issue (need mmaps re-extracting). Thanks you ;)

**Comment by DDuarte** [comment]
Fixed in 051e3b631db74886479c58485fad46b9a326a966
